cmd/adapter/adapter.go

```diff
@@ -81,6 +95,9 @@ type PrometheusAdapter struct {
 	// MetricsMaxAge is the period to query available metrics for
 	MetricsMaxAge time.Duration
 
+	MetricsPort string
+	TLSConfig   string
+	MetricsHost string
```

Comment:
could you perhaps document these fields as done above?

Reply:
Done